### PR TITLE
Bulk user profile implemented

### PR DESCRIPTION
**Pull request by ijunaid**

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -265,47 +265,75 @@ class _MyAppState extends State<MyApp> {
...
   }
 
   void setOnce() {
-    Countly.setOnce('setOnce', '200');
+    Countly.instance.userProfile.setOnce('setOnce', '200');
   }
 
   void pushUniqueValue() {
-    Countly.pushUniqueValue('pushUniqueValue', 'morning');
+    Countly.instance.userProfile.pushUnique('pushUniqueValue', 'morning');
```
Comment (turtledreams) [inline example/lib/main.dart:320]: name change?

File: example/lib/main.dart [hunk 2]
```diff
@@ -265,47 +265,75 @@ class _MyAppState extends State<MyApp> {
...
   }
 
   void pushValue() {
-    Countly.pushValue('pushValue', 'morning');
+    Countly.instance.userProfile.push('pushValue', 'morning');
```
Comment (turtledreams) [inline example/lib/main.dart:324]: here as well?

File: example/lib/main.dart (hunk 1 above)
Comment (ArtursKadikis) [reply example/lib/main.dart:320]: that would be inline with the android names

File: example/lib/main.dart (hunk 2 above)
Comment (ArtursKadikis) [reply example/lib/main.dart:324]: that would be inline with the android names

**Review by turtledreams — APPROVED** [review]

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
Let's update the deprecation messages to be more specific

**Review by ArtursKadikis — APPROVED** [review]

**Pull request merged**